hummingbird/ml/operator_converters/onnx/onnx_operator.py

```diff
@@ -64,15 +64,18 @@ class Sum(PhysicalOperator, torch.nn.Module):
     def __init__(self, logical_operator):
         super(Sum, self).__init__(logical_operator)
 
-    def forward(self, x):
-        return torch.sum(x).view(1)
+    def forward(self, *x):
+        if len(x) > 1:
+            x = torch.cat(x, dim=1)
+        return torch.sum(*x)
 
 
 class Add(PhysicalOperator, torch.nn.Module):
     def __init__(self, logical_operator, val):
         super(Add, self).__init__(logical_operator)
 
-        self.val = torch.nn.Parameter(torch.FloatTensor(val), requires_grad=False)
+        if val is not None:
```

Comment:
Can we have an assert here that checks that `len(self.inputs) == 1` when `val is not None`? We need to make sure that indeed it will go in the right branch inside `forward`